diht/model_zoo.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import logging
+import os
+import urllib
+from pathlib import Path
```

Comment:
I think code in this file might have been referenced from somewhere else. Lets put appropriate link if so.